Settings.StyleCop

```diff
@@ -117,6 +163,61 @@
             <BooleanProperty Name="Enabled">False</BooleanProperty>
           </RuleSettings>
         </Rule>
+        <Rule Name="StaticElementsMustAppearBeforeInstanceElements">
+          <!-- it's OK to have static members after instance members -->
+          <RuleSettings>
+            <BooleanProperty Name="Enabled">False</BooleanProperty>
+          </RuleSettings>
+        </Rule>
+      </Rules>
+      <AnalyzerSettings />
+    </Analyzer>
+    <Analyzer AnalyzerId="StyleCop.CSharp.LayoutRules">
+      <Rules>
+        <Rule Name="ClosingCurlyBracketMustBeFollowedByBlankLine">
+          <!-- it's OK to not have a blank line after '}' -->
```

Comment:
(!) I'd do not turn out the rule. I believe it makes a code more readable.

Reply:
Almost all occurrences are in the following form:
```
    ...
}
return ...
```

I don't think it's worthwhile to fix them.

Reply:
I tend to agree with Ilya. There's certainly no need to fix _every example_ in a hurry if that is a large work item, but it does tend to detract from readability when you end up stacking block statements adjacent to one another with individual lines woven between them.

It doesn't _always_ merit a fix, but I think having it be pointed out as standard is valuable nonetheless. Is there a way we can configure it to register as a lower-level warning or something, that doesn't itself tend to affect scoring but is still called out when it occurs?

Reply:
> Is there a way we can configure it to register as a lower-level warning or something, that doesn't itself tend to affect scoring but is still called out when it occurs

I wish there was such a configuration to StyleCop :)

Reply:
I prefer to have this enabled.  I'll see if I can just script this one and submit as a PR.

Reply:
@iSazonov could be worth trying.  Let me see about scripting this and see what the resulting commits look like.

Reply:
https://github.com/PowerShell/PowerShell/pull/8530

Reply:
@SteveL-MSFT That's a huuuuuuuge PR 👍 
I will keep this rule enabled.